src/Context/FeatureContext.php

```diff
@@ -693,8 +700,31 @@ public function __construct() {
 	 */
 	public function get_command_with_coverage( $cmd ) {
 		$with_code_coverage = (string) getenv( 'WP_CLI_TEST_COVERAGE' );
+
 		if ( \in_array( $with_code_coverage, [ 'true', '1' ], true ) ) {
-			return preg_replace( '/(^wp )|( wp )|(\/wp )/', '$1$2$3--require={SRC_DIR}/utils/generate-coverage.php ', $cmd );
+
+			$modify_command = function ( $part ) {
+				if ( preg_match( '/(^wp )|( wp )|(\/wp )/', $part ) ) {
+					$part = preg_replace( '/(^wp )|( wp )|(\/wp )/', '$1$2$3', $part );
```

Comment:
@mrsdizzie Just to clarify, does this now work with aliases?

Reply:
yes -- what fixes the previous alias problem is putting the --require at the end of the wp command instead of the beginning (since an alias must be the first thing after wp). All this extra code is needed because you can't just stick --require at the end of the full command because it might look like `wp eval xyz | something else` so you need to split on pipes (if present) and make sure its only put on the end of a wp command

Reply:
Ah I see it now. Tested with `features/aliases.feature` and it works as expected there.

Path resolution in wp-cli-tests itself works too, even within wp-cli-dev.

So I think this is good to go. You would need to approve it though :)